recipes/c-client/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def requirements(self):
+        if not self._is_msvc:
+            self.requires("openssl/1.1.1m")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True)
+
+    def _patch_msvc(self):
+        opt_flags = "/O2 /Ob2 /DNDEBUG"
+        if self.settings.build_type == "Debug":
+            opt_flags = "/Zi /Ob0 /Od /RTC1"
+        runtime = f"/{self.settings.compiler.runtime}"
+        # NOTE: boatloads of warnings for truncation, sign mismatch,
+        #       implicit conversions, just the usual C things
+        warnings = \
+            "/W3 /wd4267 /wd4244 /wd4273 /wd4311 /wd4312 /wd4133 /wd4028"
+        cflags = f"{runtime} {warnings} /GS {opt_flags}"
+        search = "EXTRACFLAGS ="
+        replace = f"EXTRACFLAGS = {cflags}"
+        tools.replace_in_file(r"src\osdep\nt\makefile.w2k", search, replace)
+
+    def _build_msvc(self):
+        make = "nmake /nologo /f makefile.w2k"
+        with tools.vcvars(self):
+            self.run(f"{make} c-client")
+            self.run(make, cwd="c-client")
+
+    def _chmod_x(self, path):
+        os.chmod(path, os.stat(path).st_mode | stat.S_IEXEC)
+
+    def _build_unix(self):
+        with open("ip6", "a"): pass
+        self._chmod_x("tools/an")
+        self._chmod_x("tools/ua")
+        unix = "src/osdep/unix"
+        self._chmod_x(f"{unix}/drivers")
+        self._chmod_x(f"{unix}/mkauths")
+        search = "SSLDIR=/usr/local/ssl"
+        ssldir = self.deps_cpp_info["openssl"].rootpath
+        tools.replace_in_file(f"{unix}/Makefile", search, f"SSLDIR={ssldir}")
+        # This is from the Homebrew Formula
+        tools.replace_in_file(
+            "src/osdep/unix/ssl_unix.c",
+            "#include <x509v3.h>\n#include <ssl.h>",
+            "#include <ssl.h>\n#include <x509v3.h>"
+        )
+        target = "osx" if self.settings.os == "Macos" else "slx"
+        args = ["IP=6", "-j1"]
+        AutoToolsBuildEnvironment(self).make(target=target, args=args)
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        if self._is_msvc:
+            self._patch_msvc()
+            self._build_msvc()
+        else:
+            self._build_unix()
+
+    def package(self):
+        self.copy("LICENSE.txt", "licenses")
+        if self._is_msvc:
+            self.copy("c-client/*.h", "include")
+            self.copy("*.lib", "lib", "c-client")
+        else:
+            self.copy("c-client/*.h", "include")
+            self.copy("*.a", "lib", "c-client")
+
+    def package_info(self):
+        if self._is_msvc:
+            self.cpp_info.system_libs = \
+                ["Winmm", "Ws2_32", "Secur32", "Crypt32"]
+        else:
+            self.cpp_info.defines = ["_DEFAULT_SOURCE"]
+            self.cpp_info.system_libs = ["crypt"]
```

Comment:
I think you want libgcrypt, correct? Then use https://conan.io/center/libgcrypt instead.

Reply:
it's part of glibc https://man7.org/linux/man-pages/man3/crypt.3.html

Reply:
Right. Doing a quick search, on Ubuntu, it's packaged separated: https://launchpad.net/ubuntu/+source/libxcrypt/ 
We could package it too ...

Reply:
Does this require further action on my part?